### PR TITLE
Update task-configuration-file.md

### PR DESCRIPTION
**Pull request by pepezzzz**
add full backup ftwrl usage details

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add full backup ftwrl usage details.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/task-configuration-file.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ mysql-instances:
     - `incremental`：只通过 binlog 把上游数据库的增量修改复制到下游数据库, 可以设置实例配置的 `meta` 配置项来指定增量复制开始的位置。
     - `all`：`full` + `incremental`。先全量备份上游数据库，将数据全量导入到下游数据库，然后从全量数据备份时导出的位置信息 (binlog position) 开始通过 binlog 增量复制数据到下游数据库。
 
+> **注意：**
+>
+> DM 2.0 使用 dumpling 工具执行全量备份。使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，保证并发备份连接的全局一致性和记录增量复制所需的起始 pos 信息，所有的并发备份连接启动事务后释放该锁。推荐在业务低峰或者 MySQL 备库上进行全量备份。
```
Comment (CharLotteiu) [inline zh/task-configuration-file.md:75]:
```suggestion
> DM 2.0 使用 dumpling 工具执行全量备份。全量备份过程中会使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，保证并发备份连接的全局一致性和记录增量复制所需的起始 pos 信息。所有的并发备份连接启动事务后释放该锁。
> 推荐在业务低峰或者 MySQL 备库上进行全量备份。
```

**Review by CharLotteiu — APPROVED** [review]
LGTM

File: zh/task-configuration-file.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ mysql-instances:
     - `incremental`：只通过 binlog 把上游数据库的增量修改复制到下游数据库, 可以设置实例配置的 `meta` 配置项来指定增量复制开始的位置。
     - `all`：`full` + `incremental`。先全量备份上游数据库，将数据全量导入到下游数据库，然后从全量数据备份时导出的位置信息 (binlog position) 开始通过 binlog 增量复制数据到下游数据库。
 
+> **注意：**
+>
+> DM 2.0 使用 dumpling 工具执行全量备份。全量备份过程中会使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，保证并发备份连接的全局一致性和记录增量复制所需的起始 pos 信息。所有的并发备份连接启动事务后释放该锁。
```
Comment (lance6716) [inline zh/task-configuration-file.md:75]:
```suggestion
> DM 2.0 使用 dumpling 工具执行全量备份。全量备份过程中会使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，从而保证并发备份连接的一致性并记录 binlog 位置用于增量复制。所有的并发备份连接启动事务后释放该锁。
```

**Review by lance6716 — APPROVED** [review]
lgtm

File: zh/task-configuration-file.md (hunk 2 above)
Comment (CharLotteiu) [reply zh/task-configuration-file.md:75]:
```suggestion
> DM 2.0 使用 dumpling 工具执行全量备份。全量备份过程中会使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作、保证备份连接的全局一致性和记录 POS 信息。所有的并发备份连接启动事务后释放该锁。
```

**Comment by CharLotteiu** [comment]
@pepezzzz Please apply the comment~ Thx!

**Pull request merged**